### PR TITLE
Adding four recently joined faculty to MBZUAI

### PR DESCRIPTION
**Pull request by PJPurnell**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

0. Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv"). **IF YOU DO NOT DO THIS, YOUR COMMIT WILL BE SUMMARILY REJECTED.**

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

2. _Insert new faculty **in alphabetical order**, not at the end of `csrankings-[a-z].csv`._ **Do not modify `csrankings.csv`, which is auto-generated.** **DO NOT MODIFY `csrankings-[0-9].csv`, WHICH ARE BEING DEPRECATED. USE `csrankings-[a-z].csv` INSTEAD.**

3. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [ x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [ x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [ x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**; please **do not** use the numeric suffixes, which are being obsoleted); include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x ] If the institution you are adding is not in the US,
update `country-info.csv`.

**(Advanced) Quick contribution via a shallow clone** 

A full clone of the CSrankings repository is almost 2GB.  To contribute a change without creating a full local clone of the CSrankings repo, you can do a _shallow clone_.  To do so, follow these steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)
2. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.
3. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to do another contribution and some time has passed, perform steps 1-3 again, creating a fresh fork and shallow clone.

### CODE REVIEW

File: csrankings-k.csv [hunk 1]
```diff
@@ -833,7 +833,7 @@ Kun Sun 0001,George Mason University,https://csis.gmu.edu/ksun,u18hG4gAAAAJ
 Kun Xie 0001,Hunan University,https://cskxie.github.io,2cfH6_0AAAAJ
 Kun Xu 0003,Tsinghua University,http://cg.cs.tsinghua.edu.cn/people/~kun,dRlBzscAAAAJ
 Kun Yang 0001,University of Essex,https://www.essex.ac.uk/people/yangk82907/kun-yang,qpe3iUkAAAAJ
-Kun Zhang 0001,Carnegie Mellon University,https://www.cmu.edu/dietrich/philosophy/people/faculty/zhang.html,RGoypN4AAAAJ
+Kun Zhang 0001,MBZUAI,https://mbzuai.ac.ae/study/faculty/dr-kun-zhang,RGoypN4AAAAJ
```
Comment (emeryberger) [inline csrankings-k.csv:836]: Website says "Kun Zhang has joined MBZUAI as Associate Professor in the Machine Learning Department. He also maintains an associate professorship at Carnegie Mellon University." Please verify that this is in fact a full-time appointment at MBZUAI (CMU page doesn't indicate change to MBZUAI affiliation).

File: csrankings-q.csv [hunk 2]
```diff
@@ -103,6 +103,7 @@ Qinpei Zhao,Tongji University,https://sse.tongji.edu.cn/Data/View/2821,NOSCHOLAR
 Qinping Zhao,Beihang University,http://scse.buaa.edu.cn/info/1084/2894.htm,NOSCHOLARPAGE
 Qinru Qiu,Syracuse University,http://hydrogen.syr.edu/~qqiu,DWv0xG0AAAAJ
 Qiong Luo 0001,HKUST,https://www.cse.ust.hk/~luo,SqAsppUAAAAJ
+Qirong Ho,MBZUAI,https://mbzuai.ac.ae/study/faculty/dr-qirong-ho,tR3AZbwAAAAJ
```
Comment (emeryberger) [inline csrankings-q.csv:106]: Employed as CTO of Petuum; seems incompatible with a full-time academic appointment. Please clarify.

File: csrankings-h.csv [hunk 3]
```diff
@@ -266,7 +266,7 @@ Hao Chen 0003,Univ. of California - Davis,http://web.cs.ucdavis.edu/~hchen,1Aa3q
 Hao Chen 0011,HKUST,https://cse.hkust.edu.hk/~jhc,Z_t5DjwAAAAJ
 Hao Dong 0003,Peking University,https://zsdonghao.github.io,xLFL4sMAAAAJ
 Hao Li 0011,Xi'an Jiaotong University,https://aquatoney.github.io,-rpSVLEAAAAJ
-Hao Li 0015,University of Southern California,http://hao-li.com,NFeigSoAAAAJ
+Hao Li 0015,MBZUAI,https://mbzuai.ac.ae/study/faculty/hao-li,NFeigSoAAAAJ
```
Comment (emeryberger) [inline csrankings-h.csv:269]: CEO of a startup (Pinscreen); seems incompatible with a full-time faculty position. Please clarify.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-h.csv (hunk 3 above)
Comment (PJPurnell) [reply csrankings-h.csv:269]: Please see official confirmation letter from MBZUAI HR Director as an email attachment

File: csrankings-k.csv (hunk 1 above)
Comment (PJPurnell) [reply csrankings-k.csv:836]: Please see official verification in letter from MBZUAI HR Director as an email attachment

File: csrankings-q.csv (hunk 2 above)
Comment (PJPurnell) [reply csrankings-q.csv:106]: Please see official confirmation in letter from MBZUAI HR Director as an email attachment

File: csrankings-h.csv (hunk 3 above)
Comment (PJPurnell) [reply csrankings-h.csv:269]: Please see official confirmation in letter from MBZUAI HR Director as an email attachment

File: csrankings-k.csv (hunk 1 above)
Comment (PJPurnell) [reply csrankings-k.csv:836]: Please see official confirmation in letter from MBZUAI HR Director as an email attachment

File: csrankings-q.csv (hunk 2 above)
Comment (PJPurnell) [reply csrankings-q.csv:106]: Please see official confirmation in letter from MBZUAI HR Director as an email attachment

**Comment by PJPurnell** [comment]
I have sent the required confirmation of the six faculty members in this pull request as an attachment in an email

**Review by PJPurnell — COMMENTED** [review]
I have sent an official letter confirming these faculty members from MBZUAI HR Director as an email attachment

**Comment by emeryberger** [comment]
Attached here for reference.

[MBZUAI Letter to CS Ranking Committee.pdf](https://github.com/emeryberger/CSrankings/files/8872752/MBZUAI.Letter.to.CS.Ranking.Committee.pdf)

**Comment by PJPurnell** [comment]
Dear Prof. Emery,

Can you please advise on which date we can expect the next CSRankings update? I understand the last DLBP import was 25th March.

With many thanks in advance and best wishes,
Philip


Philip Purnell
Head of Institutional Research and Benchmarking

T: +971 2 811 3226
E: ***@***.***
www.mbzuai.ac.ae<http://www.mbzuai.ac.ae>

Mohamed bin Zayed University of Artificial Intelligence
Masdar City, Building 1B, 3rd Floor
Abu Dhabi, UAE

[cid:MBZUAI_Logo_4d283c4f-1a0d-4aed-8a25-b17ff8b431f8.png]

The world’s first specialized research-based AI university
From: Emery Berger ***@***.***>
Date: Thursday, 9 June 2022 at 9:58 PM
To: emeryberger/CSrankings ***@***.***>
Cc: Philip Purnell ***@***.***>, Author ***@***.***>
Subject: Re: [emeryberger/CSrankings] Adding four recently joined faculty to MBZUAI (PR #4636)

Attached here for reference.

MBZUAI Letter to CS Ranking Committee.pdf<https://are01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Femeryberger%2FCSrankings%2Ffiles%2F8872752%2FMBZUAI.Letter.to.CS.Ranking.Committee.pdf&data=05%7C01%7Cphilip.purnell%40mbzuai.ac.ae%7Ceb8754461f394f75a6b808da4a418389%7Cc93272d31b074b3da3b619b34a973915%7C0%7C0%7C637903943295197941%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=HsBbGOU1dQnmnAih6LSfDSiU7C67RYds7lyfRn4E3N4%3D&reserved=0>

**Comment by emeryberger** [comment]
Updated with https://github.com/emeryberger/CSrankings/commit/4f310c5687b88e28ac56a5e4a90de55e48c2cd2e.

**Comment by PJPurnell** [comment]
Thanks for the update.

Best wishes,
Philip


Philip Purnell
Head of Institutional Research and Benchmarking

T: +971 2 811 3226
E: ***@***.***
www.mbzuai.ac.ae<http://www.mbzuai.ac.ae>

Mohamed bin Zayed University of Artificial Intelligence
Masdar City, Building 1B, 3rd Floor
Abu Dhabi, UAE

[cid:MBZUAI_Logo_4d283c4f-1a0d-4aed-8a25-b17ff8b431f8.png]

The world’s first specialized research-based AI university
From: Emery Berger ***@***.***>
Date: Monday, 11 July 2022 at 2:54 AM
To: emeryberger/CSrankings ***@***.***>
Cc: Philip Purnell ***@***.***>, Author ***@***.***>
Subject: Re: [emeryberger/CSrankings] Adding four recently joined faculty to MBZUAI (PR #4636)

Updated with 4f310c5<https://are01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Femeryberger%2FCSrankings%2Fcommit%2F4f310c5687b88e28ac56a5e4a90de55e48c2cd2e&data=05%7C01%7Cphilip.purnell%40mbzuai.ac.ae%7C7f240e5b1d6e43110ab008da62c72ef4%7Cc93272d31b074b3da3b619b34a973915%7C0%7C0%7C637930904881493869%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=vqK3Pje7bwG9FtSC0D%2Bv95jG60xKkggde8lpM8q%2BP3s%3D&reserved=0>.

**Pull request merged**